### ISSUE TITLE
Can I use Redux to make an account page in Multisite?

### ISSUE THREAD

**Issue by ChrisPlaneta** [reduxframework/redux-framework#1074]
Hi there,

I'm posting this question here, because I know no other place where I can do this. I hope you will help me, though.

I'm setting up a WP Multisite where individual blog owners get a default role of editor with heavily changed capabilities (apart from publishing and editing pages they can switch themes and that's basically it). Each theme has its own customisation options that I handle with WP customizer.

The thing is that there are some data, e.g. social links, site options, address visible to visitors, etc. that need to stay unchanged after a blog owner switches the theme (when it's set in the customizer, the data needs to be input again after changing themes). 

In other words I need to build an "account" admin page where blog owners with certain capabilities could input and save data once and it will remain there after switching themes. Is it possible? If yes, than can you give me some quick quidelines? It would save me a few hours.

Thank you for your time,
Chris
